### ISSUE TITLE
Some questions about loss function

### ISSUE THREAD

**Issue by Cathy-ye** [jfzhang95/pytorch-deeplab-xception#121]
I wonder what does each dimension of the input parameter 'logit' in loss function mean.
Code is as following
```
def CrossEntropyLoss(self, logit, target):
n,c,w,h =logit.size()
...
```

what do n c h w stand for?

**Comment by brstar96** [comment]
> I wonder what does each dimension of the input parameter 'logit' in loss function mean.
> Code is as following
> 
> ```
> def CrossEntropyLoss(self, logit, target):
> n,c,w,h =logit.size()
> ...
> ```
> 
> what do n c h w stand for?

it means n = batch size, c = channel, h = height, w = width.
note that pytorch's channel order is NCHW.

**Comment by Cathy-ye** [comment]
oh I understand,thanks!
